fe/spark-dpp/src/main/java/com/starrocks/load/loadv2/dpp/SparkDpp.java

```diff
@@ -702,6 +702,9 @@ public boolean validateData(Object srcValue, EtlJobConfig.EtlColumn etlColumn, C
                             // append columns from path to the tail
                             List<String> columnAttributes = new ArrayList<>();
                             columnAttributes.addAll(Arrays.asList(attributes));
+                            List<String> columnValueFromPath = DppUtils.parseColumnsFromPath(record
+                                            .getAs("_starrocks_internal_input_file_path"),
+                                    fileGroup.columnsFromPath);
```

Comment:
The performance may be bad if apply function DppUtils.parseColumnsFromPath row by row.

Reply:
https://github.com/StarRocks/starrocks/blob/main/fe/spark-dpp/src/main/java/com/starrocks/load/loadv2/dpp/SparkDpp.java#L855

Is it better to convert * to the real path when getting file statistics?

Reply:
Thanks for your advice.
fixed, PTAL.